react/features/welcome/components/WelcomePage.web.js

```diff
@@ -204,7 +204,9 @@ class WelcomePage extends AbstractWelcomePage {
      */
     _renderFeature(index) {
         return (
-            <div className = 'feature_holder'>
+            <div
+                className = 'feature_holder'
+                key = { index }>
```

Comment:
The WelcomePage has nothing to do with redux-logger. Please remove the modification.